docs/source/physical_robot_installation_guide/index.rst

```diff
@@ -0,0 +1,109 @@
+===================================
+Setting up a new modular robot core
+===================================
+This tutorial is for maintainers from the CI Group who want to set up a new modular robot core.
+If you simply want to know how to use a pre-installed physical robot, check the examples. This is most likely the case if you are a student.
+
+For anyone outside of the CI Group, please look at the v2 modular robot.
+
+-----------
+Preparation
+-----------
+Firstly make sure to have all materials necessary to build the physical robot.
+For **CI-Group:**
+Models to print the various Modules can be found on the following git repo: `<https://github.com/ci-group/revolve-models>`_.
+
+--------------------
+Install the Hardware
+--------------------
+For any type of hardware there are some required items:
+
+* A controller such as a RPi.
+* A HAT for the RPi to connect servos and other extensions to.
+* Modules, Servos and extensions that have to be mounted.
+* Cables and a Battery
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Hardware setup for a V1 Robot
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+**This is only applicable to the CI-Group specific V1 design.**
+Print the Modules and install the RPi in the Head compartment.
+Assemble the robot and connect the Servos to the HAT.
+More instructions can be found in the *how-to-build-a-modular-robot-guide*.
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Hardware setup for a V2 Robot
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+**This is only applicable to the CI-Group specific V2 design.**
+To build a V2 robot print all the necessary Modules and mount the RPi in the Head Compartment.
+For instructions about the HAT and the Robot architecture itself refer to `<https://github.com/ci-group/robohat>`_.
+Assemble the robot and connect the Servos to the HAT.
+
+------------------
+Setting up the RPi
+------------------
+This step is the same for all types of hardware.
+To setup the RPI follow the steps below carefully.
+
+#. Flash the SD-card with the RPi OS (previously Raspbian). Some Important notes:
+    * Reminder for CI Group people: if you select a keyboard layout, pick `English(US)`. It can always be changed using the `raspi-config` tool on the RPi.
+
+    * When setting up the WIFI connection, select your Country. (CI-Group WIFI: *ThymioNet*)
+
+#. Boot the RPi and get CLI-access. Here are some general Tips:
+    * You can connect a keyboard and screen.
+    * Better is to SSH. This work only if you and the RPi are on the same network. For CI Group, connect it to *ThymioNet* Wi-Fi.
+    * If you want to SSH and dont know the IP of the RPi use: :code:`sudo nmap -sP <your ip>` to find all clients on your network.
+
+---------------------------
+Install Revolve2 on the RPi
+---------------------------
+Setting up Revolve2 on the robot requires different steps, depending on the hardware version. Some general steps for all versions:
```

Comment:
This first sentence appears bold in my rendering and I would like it to not be bold. But I don't see why it renders as bold? You didn't put any stars around it. weird

Reply:
i know, i tried to get rid of it but  didnt find a way yet